async-aggregators/src/control_panel.py

```diff
@@ -0,0 +1,225 @@
+from typing import Any
+
+from PyQt6.QtCore import Qt, pyqtSignal
+from PyQt6.QtGui import QMouseEvent
+from PyQt6.QtWidgets import (
+    QComboBox, QFrame, QHBoxLayout, QLabel, QPushButton, QScrollArea, QSlider,
+    QVBoxLayout, QWidget
+)
+
+from src.utils.q_custom_slider import QCustomSlider
+
+
+class NoDragSlider(QSlider):
+    """QSlider subclass to disable dragging."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        super(NoDragSlider, self).__init__(*args, **kwargs)
```

Comment:
This is the old way of calling `super`.
```suggestion
        super().__init__(*args, **kwargs)
```